.gitignore

```diff
@@ -3,4 +3,4 @@
 backend/env
 **/__pycache__
 .DS_Store
-**/django.env
+.env
```

Comment:
Does this rule apply for all .env files in any subfolder? Or does it only ignore the .env files in the root directory?

Reply:
I am not so sure about it but I have changed it to **/.env.